### PR TITLE
Add a linux compatible compose file

### PR DESCRIPTION
**Pull request by JamesGardiner**
### What is the context of this PR?
Adds a Linux specific compose file and supporting Makefile targets.

### How to review 
Using a Linux host, run `make dev-compose-up-linux`, once everything successfully starts, run `make dev-compose-down-linux`.

### CODE REVIEW

**Comment by jonnyshaw89** [comment]
How do I test this? 😆

**Comment by JamesGardiner** [comment]
@jonnyshaw89 🍓🥧

**Comment by JamesGardiner** [comment]
*raspberry pi???

**Comment by JamesGardiner** [comment]
Linux VM...

**Review by bitdivision — APPROVED** [review]
Works for me

File: docker-compose-dev-linux.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    image: knarz/datastore-emulator
+    ports:
+      - "8432:8432"
+
+  redis:
+    image: redis:4
+    ports:
+      - "6379:6379"
+
+  go-launch-a-survey:
```
Comment (ajmaddaford) [inline docker-compose-dev-linux.yml]: This should be `eq-questionnaire-launcher` rather than `go-launch-a-survey`

**Review by ajmaddaford — CHANGES_REQUESTED** [review]

**Review by ajmaddaford — APPROVED** [review]

**Pull request merged**